articles/introduction-to-extended-reality/index.md

```diff
@@ -0,0 +1,73 @@
+
+![Virtual Reality](/engineering-education/introduction-to-extended-reality/virtual-reality.jpg)
+
+[Image Source: Magic Holo](https://magic-holo.com/wp-content/uploads/2017/04/Virtual_Reality.jpg)
+
+### Augmented Reality (AR)
+In this component of XR, smart devices are used to overlay sounds and digital graphics into the real world. Some of the digital details overlaid into the real-world environment include images, animation and texts. This enhanced experience can be achieved using AR screens, tablets, smartphones or screens. Snapchat filters and Pokeman Go are good examples of AR technology. In the game of Pokemon Go, digital creatures are overlaid into the real world. Snapchat filters involve putting digital objects such as glasses and hats onto the user’s head to provide unique experiences.
+
+![Augmented Reality](/engineering-education/introduction-to-extended-reality/augmented-reality.jpeg)
+
+[Image Source: Miro](https://miro.medium.com/max/1200/1*3F8SgP5Im17Kq9yPvy5rpA.jpeg)
+
+### Mixed Reality (MR)
+Mixed Reality encompasses the co-existence of real-world and digital objects. There is an interaction of these objects in real time. This is an advanced immersive technology that needs an MR headset and higher processing power than virtual reality and augmented reality. A good example of mixed reality is Microsoft's HoloLens, which allows users to interact with digital objects that are placed in a room.
+
+## How Extended Reality Works
+From a developer perspective, the extended reality is achieved by designing immersive logarithms. Software developers and data scientists should design clever and efficient algorithms to produce interesting XR features that offer unique user experiences. Other important concepts that need to be incorporated by developers of immersive technologies include 3D modelling and advanced computer vision. Artificially intelligent programs are made using machine learning, biomechanical modelling and motion tracking. Some of the programming languages used in the design of extended reality include C, C++, Java, JavaScript, Python and Swift. Full description of these languages and their application to AR can be found [here](https://howtocreateapps.com/programming-languages-ar/). Developers that may be interested in applying these programming languages in the development of VR can get more information [here](https://engineering.eckovation.com/programming-languages-develop-virtual-reality-apps/#:~:text=But%20here%20are%20four%20programming%20language%20you%20can,it%20has%20a%20huge%20...%20More%20items...%20).

```

Comment:
Divide this into two paragraphs. The second para should start from-> some of the programming languages...

Reply:
Done